src/libs/ajax.js

```diff
@@ -170,6 +171,31 @@ const User = signal => ({
       `${await Config.getTosUrlRoot()}/user/response`,
       _.mergeAll([authOpts(), { signal, method: 'POST' }, jsonBody({ ...tosData, accepted: true })])
     )
+  },
+
+  createSupportRequest: async ({ name, email, currUrl, subject, type, description }) => {
+    const options = {
+      method: 'POST',
+      body: JSON.stringify({
```

Comment:
fyi: The `jsonBody` helper will take care of the stringify and the content-type header.